Sources/MongoSwift/MongoClient.swift

```diff
@@ -252,12 +236,50 @@ public class SyncMongoClient {
         )
     }
 
+    /// This type is not meant to be instantiated directly. Should only be instantiated via subclasses.
+    // swiftformat:disable:next redundantFileprivate
+    fileprivate init(stealing pointer: OpaquePointer) {
+        self.connectionPool = ConnectionPool(stealing: pointer)
+        self.encoder = BSONEncoder()
+        self.decoder = BSONDecoder()
+        self.readConcern = nil
+        self.readPreference = ReadPreference()
+        self.writeConcern = nil
+        self.notificationCenter = NotificationCenter.default
+    }
+}
+
+extension MongoClient: Equatable {
+    public static func == (lhs: MongoClient, rhs: MongoClient) -> Bool {
+        return lhs._id == rhs._id
+    }
+}
+
+/// A synchronous MongoDB Client.
+public class SyncMongoClient: MongoClient {
+    private let operationExecutor: SyncOperationExecutor = DefaultSyncOperationExecutor()
+
     /**
-     * :nodoc:
+     * Create a new client connection to a MongoDB server. For options that included in both the connection string URI
+     * and the ClientOptions struct, the final value is set in descending order of priority: the value specified in
```

Comment:
Since these docstrings are markdown, we can write \`ClientOptions\`. Jazzy might also automatically link it to the docstring for `ClientOptions` if the backticks are used.